liblangutil/Exceptions.cpp

```diff
@@ -56,6 +59,9 @@ Error::Error(
 	case Type::TypeError:
 		m_typeName = "TypeError";
 		break;
+	case Type::GeneralError:
+		m_typeName = "GeneralError";
```

Comment:
PR states to add Notice, but it also adds GeneralError?

Reply:
I had to add this because in many places the code isn't interested in the particular error, only whether it's a Warning or Error. Since there is only one Warning, one Notice, but several Errors, I added GeneralError for the cases that create such one.

Reply:
These all seem to be Standard compiler json interface errors - maybe create a new category for them?

Reply:
And please check that this does not break the interface.

Reply:
Break which interface?